src/api/test/functional/source_controller_test.rb

```diff
@@ -3786,27 +3775,30 @@ def test_config_file
 
     get '/source/home:Iggy/_config'
     assert_response :success
-    assert_equal @response.body, CONFIG_FILE_STRING_FOR_HOME_IGGY_PROJECT
+    assert_equal @response.body, File.read("test/fixtures/files/home_iggy_project_config.txt").strip
```

Comment:
I created the methods to read fixtures files at the test_helper in my PR https://github.com/openSUSE/open-build-service/pull/1453 after being merged we should change those lines to use them.
```
def self.load_fixture(path)
  File.open(File.join(ActionController::TestCase.fixture_path, path)).read()
end

def self.load_backend_file(path)
   load_fixture("backend/#{path}")
end
```